### PR TITLE
Add tools/cloud-vision-utils.

### PR DESCRIPTION
**Pull request by cfezequiel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ The tools folder contains ready-made utilities which can simpilfy Google Cloud P
 * [BigQuery Query Plan Visualizer](tools/bq-visualizer) - A web application which provides the ability to visualise the execution stages of BigQuery query plans to aid in the optimization of queries.
 * [CloudConnect](tools/cloudconnect) - A package that automates the setup of dual VPN tunnels between AWS and GCP.
 * [Cloudera Parcel GCS Connector](tools/cloudera-parcel-gcsconnector) - This script helps you create a Cloudera parcel that includes Google Cloud Storage connector. The parcel can be deployed on a Cloudera managed cluster. This script helps you create a Cloudera parcel that includes Google Cloud Storage connector. The parcel can be deployed on a Cloudera managed cluster.
+* [Cloud AI Vision Utilities](tools/cloud-vision-utils) - This is an installable
+  python package that provides support tools for Cloud AI Vision. Currently
```
Comment (EricBeach) [inline README.md]: nit: Python

File: tools/cloud-vision-utils/cloud_vision_utils/annotation_test.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+TEST_XML = os.path.join(os.path.dirname(__file__), 'test_data.xml')
+
+
+class TestAnnotation(unittest.TestCase):
+
+  def setUp(self):
+    super().setUp()
+    self.filename = TEST_XML
+
+  def test_read_from_pascal(self):
```
Comment (EricBeach) [inline tools/cloud-vision-utils/cloud_vision_utils/annotation_test.py]: need more than one test case; error cases, corner cases, etc.

File: tools/cloud-vision-utils/cloud_vision_utils/dataset_test.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+class DatasetTest(unittest.TestCase):
+
+  NUM_IMAGES = 3
+
+  def setUp(self):
+    super().setUp()
+    self.test_images = ['test{}.jpg'.format(i) for i in range(self.NUM_IMAGES)]
+
+  def test_gen_csv_from_images(self):
```
Comment (EricBeach) [inline tools/cloud-vision-utils/cloud_vision_utils/dataset_test.py]: more test cases please

File: README.md (hunk 1 above)
Comment (cfezequiel) [reply README.md]: Done.

File: tools/cloud-vision-utils/cloud_vision_utils/annotation_test.py (hunk 2 above)
Comment (cfezequiel) [reply tools/cloud-vision-utils/cloud_vision_utils/annotation_test.py]: Done.

File: tools/cloud-vision-utils/cloud_vision_utils/dataset_test.py (hunk 3 above)
Comment (cfezequiel) [reply tools/cloud-vision-utils/cloud_vision_utils/dataset_test.py]: Done.
